src/main/java/net/sf/jabref/external/push/AbstractPushToApplication.java

```diff
@@ -32,23 +33,19 @@
 /**
  * Class for pushing entries into TexMaker.
  */
-public class PushToTexmaker implements PushToApplication {
-
-    private boolean couldNotCall;
-    private boolean notDefined;
-    private JPanel settings;
-    private final JTextField texmakerPath = new JTextField(30);
-    private final JTextField citeCommand = new JTextField(30);
+public abstract class AbstractPushToApplication implements PushToApplication {
```

Comment:
Abstract class has no abstract methods to override. Is this intended? Same methods just return null. Could they be converted to abstract methods the sub-classes must implement?